### DISCUSSION TITLE
Readonly explore links

### DISCUSSION THREAD

**Discussion by farodin91** [grafana/grafana#41087] [Explore]
It would cool be able to share a quick graph created in explore with college that don't have editor rights in Grafana.

Idea: Click on shortened link an say viewable for all link owners. Anyone that opens the link can see a read only explore with the single graph.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
